src/lib_ccx/cc_bitstream.h

```diff
@@ -25,6 +24,9 @@ struct bitstream
 	// increased by the calling function, or not.
 	unsigned char *_i_pos;
 	int _i_bpos;
+#ifndef DISABLE_RUST
+	void *rust_bs;  // Pointer to Rust's Bitstream struct
```

Comment:
Could we construct / deconstruct the struct when passing to rust instead of storing a different pointer?

There are other files, where you can see the to_ctype()

Reply:
will check this.